dpuagent/ansible/dpuagent.py

```
@@ -0,0 +1,130 @@
+skip_packages = ""
+new_add = "false"
+
+# get parameter from shell
+parser = argparse.ArgumentParser(description='Deploy dpu agent to host')
+parser.add_argument('-i', type=str, help="""specify inventory host file
+                        default=/etc/ansible/hosts""")
+parser.add_argument('--private-key', type=str, help='use this file to authenticate the connection')
+parser.add_argument('-e', type=str, help='set additional variables as key=value or YAML/JSON')
+
+args = parser.parse_args()
+argument_dict = eval(args.e)
```

Ruff (0.15.2)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)